doc/sailfish-webview-webview.qdoc

```diff
@@ -66,6 +68,505 @@
+*/
+
+/*!
+  \qmlproperty bool WebView::canShowSelectionMarkers
+  \brief Whether text selection markers will be shown
+
+  If selection markers are allowed to be shown, draggable text selection
+  markers will be shown when the user selects text from a document
+  shown in the webview.
+*/
+
+/*!
+  \qmlproperty bool WebView::textSelectionActive
+  \readonly
+  \brief Whether the user is currently selecting text
+
+  While text selection is active, the text selection controller will
+  be receiving touch input events (e.g. to allow the selection markers
+  to be dragged), rather than the webview itself (thus preventing
+  accidental panning etc).
+*/
+
+/*!
+  \qmlproperty Item WebView::textSelectionController
+  \brief The item which handles text selection gestures
+
+  If the client application doesn't explicitly provide a text selection
+  controller, a default TextSelectionController from Sailfish.WebView.Controls
+  will be constructed and used when required.
+
+  Any text selection controller provided by the client must conform
+  to the required API.  See the documentation for\c{Sailfish.WebView.Controls}
+  for more information.
+*/
+
+/*!
+  \qmlmethod WebView::clearSelection()
+  \brief Clears any text currently selected by the user.
+*/
+
+/*!
+  \qmlsignal WebView::linkClicked(string url)
+  \brief This signal is emitted when the user selects a link on the page.
+
+  When emitted \a url will be set to href selected.
+*/
+
+// ------------------------------------------------------ from RawWebView.qml:
+
+/*!
+  \qmlproperty qreal WebView::virtualKeyboardMargin
+  \brief The margin taken by the keyboard at the bottom of the page
+
+  When the keybaord is opened it may overlap web content displayed in a
+  \l{WebView}. Setting this value allows the web content to be arranged to
+  compensate for the overlapped area.
+
+  When using a \l{WebView} this property will be set automatically when the
+  keyboard is opened or closed.
+*/
+
+/*!
+  \qmlsignal WebView::contentOrientationChanged(Qt::ScreenOrientation orientation)
+  \brief This signal is emitted when the screen orientation of the rendered
+         content changes.
+*/
+
+/*!
+  \internal
+
+  \qmlsignal WebView::openUrlInNewWindow()
+  \brief This signal is emitted to indicate a request to open a new window.
+*/
+
+// ------------------------------------------------------ from QuickMozView.h:
+
+/*!
+  \qmlproperty bool WebView::privateMode
+  \brief Whether the webview is in private browsing mode
+
+  If private mode is enabled, history will not be stored in the device.
+
+  Note: the value for this property must be set when the webview is
+  initialized.  Any attempt to set the value of this property after the
+  webview has been instantiated will have no effect.
+*/
+
+/*!
+  \qmlproperty bool WebView::active
+  \brief Whether the webview is actively evaluating or rendering content
+
+  If a webview is not set as active, it will not display any content.
```

Comment:
This is good... added some here as well...

https://github.com/sailfishos/sailfish-components-webview/pull/110/files

Feel free to merge.

Reply:
Took a liberty to merge this and https://github.com/sailfishos/sailfish-components-webview/pull/110

Reply:
Yep, after rebase, this has been updated